src/app/models/CharacterPresenceModel.ts

```diff
@@ -0,0 +1,45 @@
+import { Unique, Column, ObjectIdColumn, Entity, ObjectID } from "typeorm";
```

Comment:
We don't currently enforce it as Typescript isn't properly installed on the project, but please can you remove the spaces from the start and end of each import, e.g:

```suggestion
import {Unique, Column, ObjectIdColumn, Entity, ObjectID} from "typeorm";
```

This will be later enforced by eslint, so lets save the headaches now :)